docs/user-guide/certificates-configuration-questionnaire.md

```diff
@@ -0,0 +1,118 @@
+
+When the certificates are signed by a public CA, their genuineness can be easily verified due to the transitive trust policy based on the signing certificates chain and the public availability of that certificates.   
+When the CA is a private to the organization, the clients must explicitly request the CA certificate of the private Root CA and then import it to their trust stores in order to apply the transitive trust policy.   
+
+Choose the type of certificate-signing:
+- Self-signed
+- Private CA
+- Public CA
+
+4. Client or Server validation
+
+Certificates usage can be limited to a server, a client or both. This means that a certificate explicitly created for server usage, can't be used by a client application, and vice-versa a client certificate can't be used by a server component.
+If needed to use the same certificate for both usage types, then it is necessary to either not limit the usage to one type or specify both types in the Extended Usage Attribute. Zowe requires to explicitly state the Client and Server usage in the Extended Usage Atribute of the certificates.
+
+- Only Client validation
+- Only Server validation
+- Both, Client and Server validation
+
+## Certificates configuration questionnaire
+
+Now when you understand the certificates configuration options, answer the following questions
+to choose the best options for certificates type and properties according to your Zowe deployment plan:
+
+1. What is your target environment
+   1. Production - Live production systems open for access from the internet/VPN
+   2. Test (Services, Integrations, Set up, )
+   3. Zowe Development (Enhance Zowe, Develop extensions, Develop services, ...)
+   4. Private system (Learning, Experimenting)
+
+2. Will your installation run on mainframe (z/OS, USS)?
+   1. Yes, all Zowe server components will be hosted on the mainframe.
+   2. No, all Zowe server components will be installed off-platform (z/OS).
+   3. Hybrid approach - some Zowe server components will be installed on-platform and some will be installed off-platform. 
+
+3. What certificates storage type you plan to use? 
+   1. I plan to use z/OS keyring
+   2. I plan to use keystore/truststore files pair
+
+4. Was the certificate already imported to your keystore/truststore (regardless of the certificate format and storage type)
+   1. Yes, my valid certificates were previously imported
+   2. No, I need to import my new certificates
+
+5. Do you intend to use your certificate for server, for client, or for both?
+   1. Server only
+   2. Client only
+   3. Server and Client
+   
+## Certificates configuration decision flow
+Review the diagram below to understand the certificates configuration decision flow.
+Use the answers you provided in the questionnaire above to decide which path to follow in corresponding decision block (the yellow diamonds).
```

Comment:
we try to avoid the use of "above" or "below." Please replace "above" with "preceding".

Reply:
Done